packages/reports/addon/templates/components/navi-warning.hbs

```diff
@@ -0,0 +1,34 @@
+{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#if warningEnabled}}
```

Comment:
Feels like this belongs outside the component. This would render an empty div in the dom if `warningEnabled` is false.